client/src/components/tools/toolMetadata.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div>
+  <div class="col-8">
```

Comment:
try and avoid using class="col-8" as those are hard coded to be 8 grid columns wide and will break on mobile. instead, add the md- naming to that column class to make it responsive

example: <div class="col-md-8">

see full options here
https://getbootstrap.com/docs/4.0/layout/grid/

if needed, <b-container md="8"> also gives the same type of responsive layout. NOTE: <b-container> has extra margins so you may need class="m-0" depending on the exact layout you want

https://bootstrap-vue.org/docs/components/layout